### PR TITLE
Upgrade pdf.js

### PR DESCRIPTION
**Pull request by solflare**
This is now just a simple upgrade for pdf.js since we no longer need our own changes to have this work. It includes the build files that will need rebuilding the next time we upgrade.

CC @kwiens

### CODE REVIEW

File: web/viewer.css [hunk 1]
```diff
@@ -846,73 +852,83 @@ html[dir="rtl"] .secondaryToolbarButton::before {
...
 }
 
 .toolbarButton.print::before,
 .secondaryToolbarButton.print::before {
-  content: url(images/toolbarButton-print.png);
+  content: url(/static/pdfjs/web/images/toolbarButton-print.png);
+  left: 6px;
+}
+
+html[dir="ltr"] .secondaryTollbarButton.print::before {
```
Comment (sterlinghirsh) [inline web/viewer.css]: This one says Tollbar instead of Toolbar

Comment (sterlinghirsh) [inline web/viewer.css]: Also, should this have the print button like the second one? Seems weird that our rtl is more thorough than ltr

Comment (solflare) [reply web/viewer.css]: Ya I thought that was strange too but from the changes it seems like this is how it was.

**Comment by sterlinghirsh** [comment]
CR :zap: as long as we're okay with the print button

**Comment by kwiens** [comment]
Best branch name ever?

**Comment by solflare** [comment]
It might be better to have this branch completely replace the master branch since the conflicts are Crazy (capital C crazy).

**Comment by solflare** [comment]
We will want to have this branch replace master when we're ready to do it.

**Pull request merged**